test/ui_state/user_value/buffer_ph_test.py

```diff
@@ -167,7 +171,34 @@ def test_buffer_ph(print_mock, set_next_state_mock):
         ]
     )
 
+    buffer_ph.handle_key("A")
+    set_next_state_mock.assert_not_called()
+    assert buffer_ph.value == ""
+
+    buffer_ph.loop()
+    print_mock.assert_has_calls(
+        [
+            mock.call("Enter buffer pH:", line=1),
+            mock.call("", style="center", line=2),
+            mock.call("* = .       B = BS", line=3),
```

Comment:
Can we instructions? What do you think of this message:

```text
 *=. A)ccept  B)ack
     D)ecline C)lear
```

Reply:
I like it. It is implemented